runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -73,11 +79,16 @@
 	);
 
 	private static final long PRAYER_DRAIN_INTERVAL_MS = 18200;
-	private static final int CRYPT_REGION_ID = 14231;
+	static final int CRYPT_REGION_ID = 14231;
 
 	private LoopTimer barrowsPrayerDrainTimer;
 	private boolean wasInCrypt = false;
 
+	private int barrowsStartingPotential = 0;
+
+	@Getter
+	private static Map<String, Integer> monstersKilled = new HashMap<>();
```

Comment:
This can be a `Map<CryptMonster, Integer>` as the name can be fetched from the looked up monster in the infobox at render time. Additionally, this doesn't really need to be static as there should only ever be this one field present in the single instantiated plugin instance.
```suggestion
	private Map<CryptMonster, Integer> monstersKilled = new EnumMap<>(CryptMonster.class);
```